testutils/utils.go

```diff
@@ -134,9 +168,9 @@ func RunIPAMPlugin(netconf, command, args, cniVersion string) (*current.Result,
 		panic("some error found")
 	}
 
-	io.WriteString(stdin, netconf)
-	io.WriteString(stdin, "\n")
-	stdin.Close()
+	_, _ = io.WriteString(stdin, netconf)
```

Comment:
if you ignore all return parms then no need to capture them at all.

Reply:
see the other comment

Reply:
yup yup